src/app/app.module.ts

```diff
@@ -57,7 +57,8 @@ registerLocaleData(localeIt, 'it');
   providers: [
     httpInterceptorProviders,
     ScreenTrackingService,
-    UserTrackingService
+    UserTrackingService,
+    // { provide: DEBUG_MODE, useValue: true }
```

Comment:
maybe remove commented line

Reply:
I left it in as I was always struggling to find how to reactivate the debug-view again. I think I'll try to activate it dynamically on non production builds.